airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -112,7 +112,7 @@ class KubernetesPodOperator(BaseOperator):
     :param annotations: non-identifying metadata you can attach to the Pod.
         Can be a large range of data, and can include characters
         that are not permitted by labels.
-    :param resources: resources for the launched pod.
+    :param k8s_resources: resources for the launched pod.
```

Comment:
Just something to think about
Do we want k8s_ or kubernetes_ ?

Reply:
Wouldn't this be a breaking change with a significant impact on user DAGs? Curious, what is the backward compatibility philosophy for such changes?

Reply:
or what about `pod_resources`

Reply:
another option would be to accept `task_resources` and forward this to BaseOperator `resources`

Reply:
> or what about `pod_resources` -- this seems better

`container_resources` would be better (it is set on the container after all)

Reply:
ah interesting yeah sounds good

Reply:
Let’s go with `container_resources` then.

@savingoyal Passing `resources` will continue to work, emitting a deprecation warning. You must use the new argument for task mapping, but that doesn’t work right now anyway so nothing is broken.

Reply:
container_resources sounds good